### ISSUE TITLE
Picking agent on a 3d display on Windows is really slow

### ISSUE THREAD

**Issue by benoitgaudou** [gama-platform/gama#3764]
**Describe the bug**
On GAMA 1.9.1 (Windows 11 virtualized on Mac), I run the model `Traffic and Pollution.gaml` (`Toys model/Traffic`), when I try to pick an agent, the appearance of the contextual menu takes 5-6seconds, almost freezing GAMA meanwhile.

Note that on a colleague's computer, when he right-clicked many times to pick roads, he was able to make GAMA totally freeze.

**To Reproduce**
1. Open the model `Traffic and Pollution.gaml` (`Toys model/Traffic`)
2. close the fullscreen
3. right-click on an agent. The appearance of the contextual menu takes several seconds.

Note that on MacOS, the time taken to make the popup menu appears is definitely smaller.

**Desktop (please complete the following information):**
 - OS: Windows 11 (virtualized on macOS)
 - GAMA version: 1.9.1 alpha

**Comment by AlexisDrogoul** [comment]
I think it really depends on the graphics card you have. Virtualised Windows is, in that respect, a no-go. And for the other computer, I can't say for sure.

**Comment by AlexisDrogoul** [comment]
Which virtual service are you using ?

**Comment by benoitgaudou** [comment]
I use Parallels Desktop for the virtualization

**Comment by AlexisDrogoul** [comment]
But how can you use Gama on Windows ARM ? (you are on a M1, right ?) -- does it mean you run an emulated GAMA on an emulated Windows ?

**Comment by sriramab** [comment]
Yes Alexis , parallels desktop allows to run intel apps on apple chips

**Comment by benoitgaudou** [comment]
Yes indeed, I have a M1.
And just download the GAMA for windows and run it under parallels.

Benoit

**Comment by AlexisDrogoul** [comment]
OK. Maybe we can close this one ? It is already a miracle that OpenGL works in such conditions !!!

**Comment by tnguyenh** [comment]
I have exactly the same problem on a real Window (10) install with 1.9. I'll check with the release.
